boards/arm64/fvp_baser_aemv8r/board.cmake

```diff
@@ -19,3 +22,9 @@ set(ARMFVP_FLAGS
   -C bp.vis.disable_visualisation=1
   -C bp.vis.rate_limit-enable=0
   )
+
+if(CONFIG_SMP)
+  list(APPEND ARMFVP_FLAGS -C cluster0.NUM_CORES=${CONFIG_MP_NUM_CPUS})
+else()
+  list(APPEND ARMFVP_FLAGS -C cluster0.NUM_CORES=1)
+endif()
```

Comment:
nit: can you do something:
```
if(CONFIG_SMP)
  NUM_CPU = ${....}
else()
  NUM_CPU = 1
endif()
list(...)
```
```

Reply:
I remove the determined statements and assign the variable `CONFIG_MP_NUM_CPUS` to `cluster0.NUM_CORES` directly.  Maybe it can be more simple.